src/components/form-field/form-field/form-field.scss

```diff
@@ -370,6 +396,69 @@
   padding-inline-end: var(--sbb-form-field-select-inline-padding-end);
 }
 
+.sbb-form-field__input ::slotted(textarea) {
+  @include sbb.scrollbar;
+
+  position: relative;
+  resize: vertical;
+  white-space: break-spaces;
+  overflow-y: auto;
+  min-height: calc((var(--sbb-typo-line-height-body-text) * 1em));
+
+  :host([negative]) & {
+    @include sbb.scrollbar($negative: true);
+  }
+
+  :host(:is(:not([data-slot-names~='label'], [label]), [hidden-label])) & {
+    padding-block-start: calc(
+      var(--sbb-form-field-min-height) / 2 - (var(--sbb-typo-line-height-body-text) * 0.5em) - var(
+          --sbb-form-field-border-width
+        )
+    );
+  }
+}
+
+// Custom resize  handle
+
+:host([data-input-type='textarea']) .sbb-form-field__input-container {
```

Comment:
Looks nice, a little bit much of code :)

Reply:
Manuel is preparing for us a design proposal to style the resize handle 👍